### DISCUSSION TITLE
Autogenerated screens only adds list view

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#9299] [Help] [answered]
Hi there,

I noticed that when generate new screens via Autogenerated screens, I now only get the list view page. The details and new screens are missing. I'm running the latest version 2.2.9.

Any idea what I might be doing wrong, and how to fix this? Thanks in advance!
![Screenshot 2023-01-10 at 17 03 21](https://user-images.githubusercontent.com/4005405/211602040-8a0d05a3-b232-46ab-9875-59e82e43c69f.png)

**Comment by shogunpurple** [comment, marked as answer]
This is now how the auto generated screens work @ecxs - there's no need for the 3 screens, as the autogenerated screen provides a single table with the side panel which has all the same functionality, keeping your apps cleaner and with less URLs to worry about.

**Reply by ecxs** [reply]
@shogunpurple thanks, and uh ... I should have read the Dec 2022 Release notes 😁 ... https://github.com/Budibase/budibase/discussions/9191
